### DISCUSSION TITLE
New product condition types

### DISCUSSION THREAD

**Discussion by Hlavtox** [PrestaShop/PrestaShop#35945] [Ideas]
### The problem

There are not enough product conditions for common merchant scenarios.

My clients usually sell other types of products then new, used and refurbished.

You also have items damaged in shipping, B-grade items with minor manufacturing defects and products returned in 14 day law period. New, used and refurbished are not good for these.

### The solution

Adding new conditions
- Damaged - slight item damage, from shipping for example, but can be sold just fine.
- Open package - items returned in 14 day period, but new apart from having opened packaging.
- B grade - items that have some slight manufacturing defects, but can be sold just fine.

### Implementation
https://github.com/PrestaShop/PrestaShop/pull/35919

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [X] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by Prestaworks** [comment]
Isn't it better to make these dynamic? A controller where you can create your own options?

**Reply by Hlavtox** [reply]
It would, but it would be much more work that I am not willing to invest. :-)

**Reply by Prestaworks** [reply]
I guess that would take some time :)
maybe it would be possible to modify the form with a module also somehow if needed.

Anyway, don't forget to map these https://schema.org/OfferItemCondition seems to be handled in ProductLazyArray

**Reply by Hlavtox** [reply]
@Prestaworks Already done ;-)
